tests/modules/vault-secrets.nix

```diff
@@ -0,0 +1,78 @@
+{ nixosPath, self, ... }@args:
+
+(import "${nixosPath}/tests/make-test-python.nix" ({ pkgs, ... }: rec {
+  name = "vault-secrets";
+  nodes = {
+    server = { pkgs, ... }:
+      let
+        serverArgs =
+          "-dev -dev-root-token-id='root' -dev-listen-address='0.0.0.0:8200'";
```

Comment:
Wow, you can just hard code the token? I want the time back that I spent trying to actually transfer the generated token...